backoffice/src/App.tsx

```diff
@@ -141,6 +166,12 @@ const App: FunctionComponent = () => {
                 <Route path={`/legacy-beacons/:id`}>
                   <SingleLegacyBeaconRecordViewWithParam />
                 </Route>
+                <Route path={`/delete/:id`}>
+                  <DeleteBeaconViewWithParam />
+                </Route>
```

Comment:
I would have preferred /beacons/delete/:id but this didn't work for some reason

Reply:
For it to work, it needs to be above 

```                <Route path={`/beacons/:id`}>
                  <SingleBeaconRecordViewWithParam />
                </Route>
                
                ```
                
                In the chain. 
                
                It's effectively hitting /beacons/:id catchall and calling that endpoint with the id being "/delete/${Id}"